README.md

```diff
@@ -1 +1,90 @@
-# mongo
\ No newline at end of file
+# mongo client
+
+[![Build Status](https://travis-ci.com/utopia-php/system.svg?branch=main)](https://travis-ci.com/utopia-php/mongo)
+![Total Downloads](https://img.shields.io/packagist/dt/utopia-php/mongo.svg)
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord)](https://appwrite.io/discord)
+
+Utopia WebSocket is a simple and lite abstraction layer around a Mongo Databse. 
```

Comment:
Description is wrong.

Reply:
ok

Reply:
It will auto-fix when we submit the library

Reply:
I'll do it